### DISCUSSION TITLE
Cannot add comments on local windows10 setup

### DISCUSSION THREAD

**Discussion by realB12** [umputun/remark42#1517] [Q&A] [answered]
I am running the r**emark42 container on my personal, local windows10 Docker-Desktop on port 5050**. When testing the frontend with http://127.0.0.1:5050/web everything is fine (because this is probably by the book and tested by the R42 team). 

However, when running a simple index.html file with the frontendsetting according to the installation instructions on a local http-server on port 8080, the page is loading fine on http://127.0.0.1:8080, anonymous login is ok but **when I want to send a new comment the server responds with HTTP-Status 500 (internal server error).** 
Any indications what's wrong or what I can do to troubleshoot this bug? How can I get more details to analyse? 
**Is my setup intended to work at all**?

**Comment by paskal** [comment, marked as answer]
Please see the remark42 container logs for hints on what went wrong.

**Reply by realB12** [reply]
Thank you very much. Have checked an got a hint that I had a typo in my docker-compose.yaml:  Had `SITE==remark8037` instead of `SITE=remark8037` resp. have used double == instead of single. Sorry for wasting your time. 
"Issue" fixed resp. identified as no issue. Thanks for help. Kind regards Rene
